app/views/admin/AdministerUserModal.js

```
@@ -287,6 +287,10 @@ module.exports = (AdministerUserModal = (function () {
       if (attrs.userTimeZone?.[0] === 'on') {
         timeZone = this.userTimeZone
       }
+      if (!attrs.revokable?.[0] !== 'on') {
+        attrs.irrevocable = true
+      }
+      delete attrs.revokable
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)